aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -116,5 +116,8 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.aggregator.destroy.base, "aggregator.destroy.base", 500 * MUL],
     [.aggregator_factory.new_aggregator.base, "aggregator_factory.new_aggregator.base", 500 * MUL],
 
-    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL]
+    [.object.exists_at.base_cost, { 7.. => "object.exists_at.base" }, 250 * MUL],
+    // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
```

Comment:
nit: ... they *are* copied ...

Reply:
@davidiw your grammar instructor Alden has a word with you XD

Reply:
I mean, I'm shocked by the quality of copying. Afaik I have nothing to do with this. And the blame agrees with me.